### ISSUE TITLE
Most Features Break... at (what seems like) random.

### ISSUE THREAD

**Issue by mcarpenterjr** [Microsoft/vscode#33856]
- VSCode Version: Code 1.15.1 (41abd21afdf7424c89319ee7cb0445cc6f376959, 2017-08-16T17:16:27.618Z)
- OS Version: Linux x64 4.10.0-33-generic
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
beautify|Hoo|1.1.1
material-icon-theme|PKi|2.2.0
code-settings-sync|Sha|2.8.3
html-css-class-completion|Zig|1.9.0
html-snippets|abu|0.1.0
vscode-database|baj|1.2.0
jshint|dba|0.10.15
jquerysnippets|don|0.0.1
vscode-html-css|ecm|0.1.8
vscode-npm-script|eg2|0.2.1
prettier-vscode|esb|0.23.1
php-debug|fel|1.11.1
php-intellisense|fel|1.5.1
code-runner|for|0.7.1
knockoutjs|fur|0.1.0
gitlab-mr|jas|1.3.0
language-polymer|jon|1.0.3
font-awesome-codes-html|med|0.0.7
prettify-json|moh|0.0.3
debugger-for-chrome|msj|3.2.1
color-highlight|nau|2.3.0
mysql-inline-decorator|odu|0.0.4
polymer-ide|pol|0.6.0
datetime|rid|1.0.5
write-good-linter|tra|0.0.3
bootstrap-3-snippets|wcw|0.1.0
JasmineSnippets|xab|1.0.0
JavaScriptSnippets|xab|1.4.1
vscode-open-in-github|ziy|1.2.7

(3 theme extensions excluded)

---

Steps to Reproduce:

1. Open a project.
  1.1 Open a new window on a second screen and drag some files from the said project if need be.
2. CODE!
3. Try to make a commit through built in SC...
4. Enter a comment, but get no response when doing `CTRL` + `ENTER` or menu Commit All.
  4.1 If you can make commit attempt to push changes.
5. Watch nothing happen.
6. Look down to see your time plugin stopped 20 minutes ago.
 6.1 Notice there is no time because you disabled the plugin go to step 9.
7. Reload the window, go to step 3.
  7.1 notice this is the second time you've had to reload the window, proceed to step 8.
8. Disable time plugin.
  8.1 Reload window, Go To step 3.
9. Completely exit code and relaunch, go to step 1.
  9.1 You've done this twice now, go to step 10.
10. Create an issue on GitHub.

Reproduces without extensions: No, not extensivley tested however.

**Comment by roblourens** [comment]
Looks like there are a couple issues here, please try to keep it to one problem per issue to make it less confusing to investigate. Also I'm not really following your steps.

> Try to make a commit through built in SC... Enter a comment, but get no response when doing CTRL + ENTER

Does this happen in any git repo? Can you check the developer tools for an error message? Any other details you can give?

> Look down to see your time plugin stopped 20 minutes ago.  Notice there is no time because you disabled the plugin go to step 9.

Why did you disable the extension? Is there an issue with VS Code or with the extension?

**Comment by mcarpenterjr** [comment]
It is hard to follow, I will admit.

I'm using git, I can make a commit through the CLI with out issue, through a standalone instance of a terminal, or through the integrated terminal in code. At random times I cannot commit through VS code itself using the SC tab.

When this issue first arose I was using a time extension, as an indicator, something was up with code, the time would be stopped. I would disable the time extension reload the window and then try committing again, this would fix the issue sometimes only for a short period of time. On occasion even after disabling the clock extension, code would still be broken. I noticed more and more things that would cease to work, emmet and IntelliSense, etc... once I had picked up on the issue.

Last night I blew out my installation and removed all of my extensions and began installing them on an as needed basis. I would speculate this is an extension issue and not a code issue at this point, not sure which extension.

**Comment by roblourens** [comment]
Ok, so you had an extension to indicate when the extension host process was no longer responsive, and that's when you had issues with git? Yeah, it sounds like maybe there was an issue with another extension. If you see it again, you might see the extensionHost process taking 100% of the CPU. If you can figure out which extension caused it, please file an issue on them.
